forms-shared/tsconfig.json

```diff
@@ -8,7 +8,8 @@
     "esModuleInterop": true,
     "skipLibCheck": true,
     "forceConsistentCasingInFileNames": true,
-    "moduleResolution": "node16"
+    "moduleResolution": "node16",
+    "jsx": "react"
```

Comment:
this should be the default - if it's important for this to be there please add it to comment / readme

Reply:
This should be default where?